app/assets/stylesheets/modern-business.scss

```diff
@@ -150,6 +150,14 @@ header.carousel .fill#slider-3 {
   font-size: 20px;
 }
 
+#index-names {
+  color: #A30505;
+}
+
+#index-titles {
+  color: #001B4D;
```

Comment:
Color literals like `#001B4D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#001B4D` should be written as `#001b4d`